### PR TITLE
Get reviewer details

### LINKED ISSUE #87: Get reviewer details API

### PR DESCRIPTION
**Pull request by thisisommore**
fix #87

### CODE REVIEW

File: api/v1/getreviewerdetails/types.go
Comment (Shachindra) [inline api/v1/getreviewerdetails/types.go:1]: Will discord and twitter handle be part of payload as well?

Comment (thisisommore) [reply api/v1/getreviewerdetails/types.go:1]: Seems like we can add them, hope in frontend somewhere mention that this fields will be visible to public. wdyt?

Comment (Shachindra) [reply api/v1/getreviewerdetails/types.go:1]: Yes.

**Review by Shachindra — APPROVED** [review]
LGTM

**Pull request merged**